js/main.js

```diff
@@ -1,3 +1,8 @@
+// Start Music
```

Comment:
@zseiman I suggest you create a separate js file solely for music. Call it `music.js` and include it in `index.html`. You can move all music related js there enclosed under a `$(function(){});` block (see `command.js').